### PR TITLE
Bah q107lm 19031

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description

As a Compare Tool User, I need to be informed of the implications of how I answer the question so that I can answer it correctly in order to receive the most accurate benefit estimates.

## Testing done
Was able to use the Herokuapp deployment from story 19115 to test the modal learn more link.

https://vagov-content-pr-484.herokuapp.com/education/about-gi-bill-benefits/post-9-11/

## Acceptance criteria
- [x] When the "Learn More" link is clicked next to the "Will the majority of your classes be on the main campus?" question, the following content is displayed:

**"Location where you'll take classes"**
VA pays monthly housing allowance (MHA) based on the campus location where you physically attend the majority of your classes.

**A campus could include:**

- A main campus: the location where the primary teaching facilities of an educational institution are located
- A branch campus: the location of an educational institution that is geographically apart from and - operationally independent of the main campus of the educational institution
- An extension campus: the location that is geographically apart from the main or branch campus but is operationally dependent on that campus for the performance of administrative tasks

**Learn more about the Location-Based Housing Allowance.**

- [x] "Location-Based Housing Allowance" in the last sentence is a link that opens the following page in a new window: https://www.va.gov/education/about-gi-bill-benefits/post-9-11/
a. The link anchors to the section "What is the Location-Based Housing Allowance (Section 107)?"
- [x] @AnneHurley and @DanielleSoCompany are tagged in the PR for these changes.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sonntag-adam — APPROVED** [review]

File: src/applications/gi/containers/Modals.jsx [hunk 1]
```diff
@@ -911,37 +912,81 @@ export class Modals extends React.Component {
...
           visible={this.shouldDisplayModal('calcBeneficiaryLocationQuestion')}
         >
-          <h3>Housing Allowance/BAH changes</h3>
-          <p>
-            Starting August 1, 2018, we'll determine the monthly housing
-            allowance under the Post-9/11 GI Bill based on the zip code of the
-            campus where you physically attend the majority of your classes. The
-            location of the school where you're enrolled won't be relevant.
-          </p>
+          {environment.isProduction() ? (
```
Comment (zurbergram) [inline src/applications/gi/containers/Modals.jsx]: pull this out into its own function

Comment (ksantiagomorgan6) [reply src/applications/gi/containers/Modals.jsx]: Completed it

File: src/applications/gi/containers/Modals.jsx [hunk 2]
```diff
@@ -28,6 +29,81 @@ export class Modals extends React.Component {
     );
   }
 
+  productionFlag107() {
```
Comment (zurbergram) [inline src/applications/gi/containers/Modals.jsx]: name should be something like `calcBeneficiaryLocationQuestionContent`

Comment (ksantiagomorgan6) [reply src/applications/gi/containers/Modals.jsx]: Updated function name to calcBeneficiaryLocationQuestionContent

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by ksantiagomorgan6** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!" as a comment to the pull request.

**Review by accbjt — APPROVED** [review]
LGTM

**Pull request merged**